distribution/deb/src/main/packaging/init.d/elasticsearch

```diff
@@ -104,6 +101,12 @@ if [ -f "$DEFAULT" ]; then
 	. "$DEFAULT"
 fi
 
+# Elasticsearch configuration file (elasticsearch.yml)
```

Comment:
Maybe a note that this was moved down here so it could use the user configured $CONF_DIR.